src/js/handler/month/creation.js

```diff
@@ -277,20 +277,21 @@ MonthCreation.prototype._onClick = function(e) {
 };
 
 /**
- * Adjust time to half hour or hour o'clock
+ * Adjust time to our o'clock
  * @param {TZDate} start - start time
  * @param {TZDate} end - end time
  * @returns {Object} start and end
  */
```

Comment:
please, add `@private` tag to private method's JSDOC

Reply:
We don't have any `@private` tag yet. All public apis are on `calendar.js` at this time.
But we will check all jsdoc in the sources.